features/admin.console-settings.v1/pages/console-settings-page.tsx

```diff
@@ -79,14 +81,16 @@ const ConsoleSettingsPage: FunctionComponent<ConsoleSettingsPageInterface> = (
                         InputProps={ {
                             endAdornment: (
                                 <Tooltip
-                                    title="Copy"
+                                    title={ isCopied ? "Copied!" : "Copy" }
                                 >
                                     <div>
                                         <IconButton
                                             aria-label="Reset field to default"
                                             className="reset-field-to-default-adornment"
                                             onClick={ async () => {
                                                 await CommonUtils.copyTextToClipboard(consoleUrl);
+                                                setIsCopied(true);
+                                                setTimeout(() => setIsCopied(false), 1000);
```

Comment:
Instead of resetting the tooltip after a fixed time, shall we make it change when user leave the copy icon, so that the user experience here will be consistent with the semantic ui based `CopyInputField`?

Reply:
> done!

I don’t see any implementation related to this improvement in the changed files though. Could you please double check if you’ve pushed the commit to the remote branch?